test/ui5-app/webapp/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8" />
+  <title>bare bones UI5 (dark 😱)</title>
+  <script id="sap-ui-bootstrap"
+          src="resources/sap-ui-core.js"
+          data-sap-ui-theme="sap_fiori_3_dark"
+          data-sap-ui-xx-cssVariables="true"
```

Comment:
using `"additional"` would be more safe for now - RTL is a bit buggy with cssVariables="true" 😅